tests/Expected/Business/testJsonSerialize.json

```diff
@@ -0,0 +1,42 @@
+{
+  "days": {
+    "1": {
+      "dayOfWeek": 1,
+      "openingIntervals": [
+        {
+          "start": {
+            "hours": 9,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 13,
+            "minutes": 0,
+            "seconds": 0
+          }
+        },
+        {
+          "start": {
+            "hours": 14,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 17,
+            "minutes": 0,
+            "seconds": 0
+          }
+        }
+      ]
+    },
+    "5": {
+      "dayOfWeek": 5,
+      "openingIntervals": []
```

Comment:
No problem @florianv .

I updated the code and added a bit of info in the readme. Waiting for the merge now.
Also no need to tag it immediately after the merge because I would like to make some more PR with some minor changes (not related to the json encoding)